### PR TITLE
elaborate "external networking"

### PR DESCRIPTION
**Pull request by vendrov**
Giving example for external netowrking

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/766)

### CODE REVIEW

File: docs/run_in_existing_vpc.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ but it's better to avoid the later confusion!)
 
 ## Running in a shared subnet
 
-You can also use a shared subnet. Doing so is not recommended unless you are using external networking.
+You can also use a shared subnet. Doing so is not recommended unless you are using external networking (weave or flannel or calico or kope-routing etc).
```
Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: Thanks for the PR

Sorry to be picky, we need to make kope-routing "external" and the others CNI. Do you want to fix that or have is fix that later? I know you did not write the initial docs

Comment (keglevich3) [inline docs/run_in_existing_vpc.md]: Is that mean that I should replace "external networking" with "CNI"?

Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: There are two different types.  You need CNI or External ... and reference the networking doc.

Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: "kope-routing" == external
others == CNI

Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: Bump

**Pull request merged**